preprocessing/feature_selection_MoranI/spatially_variable_genes_scanpy.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+# Author_and_contribution: Niklas Mueller-Boetticher; created script
+# Author_and_contribution: Qirong Mao; Implemented feature selection with Moran's I
+
+import argparse
+
+# TODO adjust description
+parser = argparse.ArgumentParser(description="SVG selection using Squidpy (Moran's I)")
+
+parser.add_argument(
+    "-c", "--coordinates", help="Path to coordinates (as tsv).", required=True
+)
+parser.add_argument(
+    "-m", "--matrix", help="Path to (transformed) counts (as mtx).", required=True
+)
+parser.add_argument(
+    "-f", "--features", help="Path to features (as tsv).", required=True
+)
+parser.add_argument(
+    "-sc", "--spatial_connectivities", help="Path to spatial connectivities (as mtx).", required=True
+)
+parser.add_argument(
+    "-sd", "--spatial_distance", help="Path to spatial distance (as mtx).", required=True
+)
+parser.add_argument(
+    "-o", "--observations", help="Path to observations (as tsv).", required=True
+)
+parser.add_argument(
+     "-n", "--n_top_genes", help="Number of genes to keep (Default:3000).", required=False, type=int, default=3000)
+
+parser.add_argument("-d", "--out_dir", help="Output directory.", required=True)
+
+parser.add_argument(
+    "--config",
+    help="Optional config file (json) used to pass additional parameters.",
+    required=False,
+)
+
+args = parser.parse_args()
+
+from pathlib import Path
+import pandas as pd
+
+out_dir = Path(args.out_dir)
+
+# Output files 
+feature_selection_file = out_dir / "features_MoranI.tsv"
```

Comment:
rename output to features.tsv to be consistent with current setup

Reply:
Done